File: frontend/src/services/Game.ts
import Player from '@/model/Player';
import { DeepReadonly, readonly, Ref } from 'vue';

export default class Game {
  public readonly players: DeepReadonly<Ref<Player[]>>;

  constructor(public readonly roomId: string, _players: Ref<Player[]>) {
    this.players = readonly(_players);
  }
}

File: backend/WerewolfCircle.Data/Game.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;

namespace WerewolfCircle.Data
{
    [Index(nameof(RoomId), IsUnique = true)]
    public class Game
    {
        public int Id { get; set; }

        [Required]
        public string RoomId { get; set; } = string.Empty;

        [Required]
        public string AdminConnectionId { get; set; } = string.Empty;

        public ICollection<Player> Players { get; set; } = null!;
    }
}
